src/main/java/com/car/app/model/BaseModel.java

```diff
@@ -31,22 +31,23 @@
 	private static final long serialVersionUID = -1399495828692808773L;
```

Comment:
Please add bean validation annotation to the entities. Both BaseModel and Car entity.